proparse/src/main/java/org/prorefactor/proparse/antlr4/ProToken.java

```diff
@@ -0,0 +1,203 @@
+/*******************************************************************************
+ * Copyright (c) 2016-2017 Gilles Querret
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Gilles Querret - initial API and implementation and/or initial documentation
+ *******************************************************************************/ 
+package org.prorefactor.proparse.antlr4;
+
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.TokenSource;
+import org.antlr.v4.runtime.WritableToken;
+
+public class ProToken implements WritableToken {
+
+  /**
+   * This is the backing field for {@link #getType} and {@link #setType}.
+   */
+  protected int type;
+
+  /**
+   * This is the backing field for {@link #getLine} and {@link #setLine}.
+   */
+  protected int line;
+
+  /**
+   * This is the backing field for {@link #getCharPositionInLine} and {@link #setCharPositionInLine}.
+   */
+  protected int charPositionInLine = -1; // set to invalid position
+
+  /**
+   * This is the backing field for {@link #getChannel} and {@link #setChannel}.
+   */
+  protected int channel = DEFAULT_CHANNEL;
+
+  /**
+   * This is the backing field for {@link #getText} when the token text is explicitly set in the constructor or via
+   * {@link #setText}.
+   *
+   * @see #getText()
+   */
+  protected String text;
+
+  /**
+   * This is the backing field for {@link #getTokenIndex} and {@link #setTokenIndex}.
+   */
+  protected int index = -1;
+
+  /**
+   * This is the backing field for {@link #getStartIndex} and {@link #setStartIndex}.
+   */
+  protected int start;
+
+  /**
+   * This is the backing field for {@link #getStopIndex} and {@link #setStopIndex}.
+   */
+  protected int stop;
+
+  private int fileIndex;
+  private int endFileIndex;
+  private int endLine;
+  private int endCharPositionInLine;
+  public int getEndFileIndex() {
+    return endFileIndex;
+  }
+
+  public void setEndFileIndex(int endFileIndex) {
+    this.endFileIndex = endFileIndex;
+  }
+
+  public int getEndLine() {
+    return endLine;
+  }
+
+  public void setEndLine(int endLine) {
+    this.endLine = endLine;
+  }
+
+  public int getEndCharPositionInLine() {
+    return endCharPositionInLine;
+  }
+
+  public void setEndCharPositionInLine(int endCharPositionInLine) {
+    this.endCharPositionInLine = endCharPositionInLine;
+  }
+
+  private int macroSourceNum;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1213)